modules/misc/qt.nix

```diff
@@ -150,6 +162,12 @@ in {
       else
         cfg.platformTheme;
       QT_STYLE_OVERRIDE = cfg.style.name;
+      QT_PLUGIN_PATH = if cfg.pluginPath != [ ] then
+        "${
+          concatStringsSep ":" cfg.pluginPath
+        }\${QT_PLUGIN_PATH:+:}$QT_PLUGIN_PATH"
+      else
+        null;
```

Comment:
Can't be null, use mkIf

Reply:
`mkIf (cfg.pluginPath != [])`? I don't know how to use mkIf here. And it said
https://github.com/nix-community/home-manager/blob/2014199bfdf72384800a6240e9b44d97a15425c5/modules/misc/qt.nix#L156-L157

Reply:
You're right, sorry.